lib/customer.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'takeaway'
+
+class Customer
+  include Takeaway
+
+  attr_accessor :order_basket, :order_total, :total_items
+
+  def initialize
+    @order_basket = {}
+    @order_total = 0
+    @total_items = 0
+  end
+
+  def view_menu
+    menu
+  end
+
+  def select_item(item, quantity)
+    fail "Sorry, that item is not on the menu." unless menu.key?(item)
+    add_to_order_basket(item, quantity)
+    update_order_total(item, quantity)
+    update_total_items(quantity)
+  end
+
+  def receipt
+    "Your order has " + total_items.to_s + " items with a total of £" + order_total.to_s
+  end
+
+  def make_payment
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for make_payment is too high. [15.07/15]